Scala/Scala.sublime-syntax

```diff
@@ -85,7 +85,7 @@ contexts:
     - match: |-
         (?x)
         \b(def)\s+
-        (([a-zA-Z$_][a-zA-Z0-9$_]*(_[^a-zA-Z0-9\s]+)?)|`.*`|[^\w\[\(\:\_\s]+)
+        (_*([a-zA-Z$][a-zA-Z0-9$]*|[^a-zA-Z0-9$`\s\(\)\[\]\{\};\.]+)(_+([a-zA-Z$][a-zA-Z0-9$]*|[^a-zA-Z0-9$`\s\(\)\[\]\{\};\._]+)?)*|`[^`]+`|[^\w\[\(\:\_\s]+)
```

Comment:
I do think that would make things more readable.  With that said, I'm planning on making a pass over this whole thing and collapsing all of the "matching the same thing, but not actually the same regex" situations that we have in the current Scala mode using variables.  When I do that, I will format the regexes in a more readable way.  However, I don't want to actually make that change until all of my PRs have landed, since I have a *lot* of pending work right now and a common-regex collapse cuts across all of it.